### ISSUE TITLE
Mega NAS permissions

### ISSUE THREAD

**Issue by mecrally** [meganz/MEGAcmd#615]
I have an issue. I will post here, then write me if I have to text a mail. I have a NAS system with a WD my cloud ex2 ultra when I connect it using megasync I receive error "MEGA can not obtain permission to write here. Do you want to continue?" so I can manage everything in my NAS but switching on mega from Firefox I can not do that... with Google Drive no issues... can you help me even paying? thanks

**Comment by mecrally** [comment]
@mecrally @polmr please can you help me? I can pay but I don't know how to solve

**Comment by polmr** [comment]
Hi @mecrally ,
First apologies for the late response.
Second, you are posting this in MEGAcmd repository, whereas you mention MEGAsync. Is MEGAsync or MEGAcmd the one you are having trouble with?
In any case, I am having trouble understanding your issue.
Where did you exactly see that?
What do you mean with "connect it"?
_so I can manage everything in my NAS but switching on mega from Firefox_ I'm not sure what you mean with that either.
The message you mention: "MEGA can not obtain permission to write here. Do you want to continue?"  is not something apparently included in any of our software either. Is that correctly typed?

**Comment by mecrally** [comment]
Hi, my issue is only for mega sync

**Comment by mecrally** [comment]
Yeah, I receive a messagge error from mega sync cause Drive support write permission from local drive and it cloud. Mega not!
Infact I have deleted some files in mega and sync deleted all files in my local drive. No problem but just to know :)

**Comment by polmr** [comment]
> No problem but just to know :)
So, let me get it straight: your synchronization established with MEGAsync is working as expected?
Let us know if otherwise, and if not, please indicate the exact error (screenshots are welcome ;-) ), and the sync configuration. 
Also bear in mind that you can always install MEGAcmd in your NAS directly: https://mega.nz/nas

**Comment by mecrally** [comment]
sure thanks! I use only mega sync and I had had an issue cause I can not use mega cloud synced instead of Drive. but no problem thanks
